### ISSUE TITLE
"Episode does not exist in Trakt!", but it does

### ISSUE THREAD

**Issue by IDeserveToBeWrited** [lukearran/TvTimeToTrakt#10]
Script worked great for most of my episodes and I'd like to thank all the contributors very much.
But for the other I've run into a weird problem where the script tells me an episode does not exist in Trakt.tv and provides a valid link for that episode.
I've modified a logging setup to log all those warnings into a file and this is what I got, maybe you'll manage to find a connection, if there is one.
[trakt.log](https://github.com/lukearran/TvTimeToTrakt/files/8543552/trakt.log)

**Comment by MohamadBayz** [comment]
I have the same problem in here, any solution?

**Comment by ak800i** [comment]
same here but I already marked them as watched so no big deal

**Comment by shackrock** [comment]
Same issue for me, occurs really randomly and right in the middle of a season for some reason.

**Comment by shackrock** [comment]
Is there a log file that shows all of the warnings in one place? Then I could at least see how many were effected.  The CMD prompt log entries (I'm in windows) are not saved (they overwrite themselves over time).

**Comment by shackrock** [comment]
Agreed, I think there's some easy improvements that could be done.  

but in the meantime, need to figure out how to correct what's completed too in these cases - at least that's my problem for now! =)

**Comment by MohamadBayz** [comment]
True
